apps/codeforafrica/src/lib/data/local/index.js

```diff
@@ -1,32 +1,14 @@
-import { getPageStaticProps } from "@/codeforafrica/lib";
 import { getPageProps } from "@/codeforafrica/lib/data/common";
```

Comment:
Outer functions can call inner functions but not the other way round i.e. `data/common` can call `data/common/local` functions but `data/common/local` shouldn't call or even be aware of `data/common` functionality.

Otherwise you end up with [circular dependencies](https://en.wikipedia.org/wiki/Circular_dependency). Again, I believe `apps/charterafrica` structure avoided all of these.